src/packages/buskill/__init__.py

```diff
@@ -224,6 +227,25 @@ def __init__(self):
 		self.CURRENT_PLATFORM = platform.system().upper()
 		self.ERR_PLATFORM_NOT_SUPPORTED = 'ERROR: Your platform (' +str(platform.system())+ ') is not supported. If you believe this is an error, please file a bug report:\n\nhttps://github.com/BusKill/buskill-app/issues'
 
+		config = SafeConfigParser()
+
+		self.CONFIG = None
+
+		# NOTE self.config will change to a dictionary if the file is found 
+		# 	if not a startup routine should begin 
+		#	Dictionary keys will be the setting and the value the variable 
+		# 	i.e. Trigger would be the key, the trigger file location would be the value 
+
+		if os.file.exists(APP_DIR, 'buskill.conf'):
+			self.CONFIG = dict()
+			config.read('buskill.conf')
+			for name, value in config.options():
+				self.config[name] == value
+		else:
+			do_something = None	
+		# NOTE This a placeholder!!!! 
+		# Here there should be a setup wizard which will create the config file (TODO Make the function to create Config files)
```

Comment:
does this suggest that currently the config file is *not* created if it doesn't yet exist?

Reply:
This is just a placeholder for now. 

On local this has been actually implemented. will be pushed after testing